src/data.rs

```diff
@@ -0,0 +1,66 @@
+use std::{
+    fs,
+    io::{Cursor, Read, Write}
+};
+use rust_intro::rust_intro::DataMessage;
+use prost::Message;
+
+const FILE: &str = "data.persist";
```

Comment:
While it is fine for this demo example, the file path should not be hardcoded here in a larger program. Instead take the path as a parameter in the `new()` method.